<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Name : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PhNum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Phon number : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Addüìù</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="family">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Family : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>440</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete üì§</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_deleteAll">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete All üóë</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_default">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Default üåû</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dark">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dark mood üåö</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>281</width>
      <height>201</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>279</width>
       <height>199</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLineEdit" name="id">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>ID : </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>299</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
